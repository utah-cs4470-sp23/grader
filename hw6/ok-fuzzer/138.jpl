show (if (! (! true)) then (! (((389 - 269) % 44) >= (- 15))) else ((if ((51.55 % 81.13) >= 21.31) then 640 else (470 + (474 - 246))) == ((if (367 == 438) then (if true then 61 else 569) else 818) + (if (! true) then 869 else 381))))

