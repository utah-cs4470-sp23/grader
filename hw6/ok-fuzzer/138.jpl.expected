(ShowCmd
 (IfExpr
  (BoolType)
  (UnopExpr (BoolType) ! (UnopExpr (BoolType) ! (TrueExpr)))
  (UnopExpr
   (BoolType)
   !
   (BinopExpr
    (BoolType)
    (BinopExpr
     (IntType)
     (BinopExpr (IntType) (IntExpr (IntType) 389) - (IntExpr (IntType) 269))
     %
     (IntExpr (IntType) 44))
    >=
    (UnopExpr (IntType) - (IntExpr (IntType) 15))))
  (BinopExpr
   (BoolType)
   (IfExpr
    (IntType)
    (BinopExpr
     (BoolType)
     (BinopExpr
      (FloatType)
      (FloatExpr (FloatType) 51)
      %
      (FloatExpr (FloatType) 81))
     >=
     (FloatExpr (FloatType) 21))
    (IntExpr (IntType) 640)
    (BinopExpr
     (IntType)
     (IntExpr (IntType) 470)
     +
     (BinopExpr (IntType) (IntExpr (IntType) 474) - (IntExpr (IntType) 246))))
   ==
   (BinopExpr
    (IntType)
    (IfExpr
     (IntType)
     (BinopExpr (BoolType) (IntExpr (IntType) 367) == (IntExpr (IntType) 438))
     (IfExpr
      (IntType)
      (TrueExpr)
      (IntExpr (IntType) 61)
      (IntExpr (IntType) 569))
     (IntExpr (IntType) 818))
    +
    (IfExpr
     (IntType)
     (UnopExpr (BoolType) ! (TrueExpr))
     (IntExpr (IntType) 869)
     (IntExpr (IntType) 381))))))
